modules/connectors/dpd_group/karrio/mappers/dpd_group/proxy.py

```diff
@@ -0,0 +1,56 @@
+"""Karrio DPD Group client proxy."""
+
+import karrio.lib as lib
+import karrio.api.proxy as proxy
+import karrio.mappers.dpd_group.settings as provider_settings
+import karrio.universal.mappers.rating_proxy as rating_proxy
+
+
+class Proxy(rating_proxy.RatingMixinProxy, proxy.Proxy):
+    settings: provider_settings.Settings
+
+    def get_rates(self, request: lib.Serializable) -> lib.Deserializable[str]:
+        return super().get_rates(request)
+
+    def _get_auth_headers(self) -> dict:
+        """Build authentication headers for DPD META-API."""
+        # access_token property returns the token string directly
+        token = self.settings.access_token
+        
+        return {
+            "Content-Type": "application/json",
+            "Authorization": f"Bearer {token}",
+        }
+
+    def create_shipment(self, request: lib.Serializable) -> lib.Deserializable[str]:
+        """
+        Create a shipment with DPD META-API.
+        POST /shipping/v1/shipment
+        """
+        # Get label format from request options or default to PDF
+        label_format = getattr(request, "label_format", "PDF") or "PDF"
```

Comment:
Use the serializer context to pass things like that. 

look at 
https://github.com/karrioapi/karrio/blob/904e5a8dfd8283a0c93948b99d57c7bdf385f859/modules/connectors/dhl_parcel_de/karrio/providers/dhl_parcel_de/shipment/create.py#L284

used later in 
https://github.com/karrioapi/karrio/blob/904e5a8dfd8283a0c93948b99d57c7bdf385f859/modules/connectors/dhl_parcel_de/karrio/mappers/dhl_parcel_de/proxy.py#L18